ports.py

```diff
@@ -1,5 +1,5 @@
 event_stream = 10001
 streamer_control = 10002
-result_stream = 10003
+result_stream = [10003, 10006, 10007, 10008]
```

Comment:
I think we should just define base ports for this, and leave big gaps between ports for different components. The streamer could then use `base + 0`, `base + 1`, etc.